Evergreen/Apps/Get-WinFSP.ps1

```diff
@@ -0,0 +1,28 @@
+Function Get-WinFSP {
```

Comment:
Please rename to `Get-NavimaticsWinFSP` - it's not likely we'll have a second app named `WinFSP`, but ideally apps should have the vendor name in the app name where possible